charts/argo-cd/values.yaml

```
@@ -1090,6 +1090,12 @@ controller:
     # -- List of custom rules for the application controller's ClusterRole resource
     rules: []
 
+  # -- Default application controller's network policy
+  networkPolicy:
```

my bad, this will be rendered in README.
```suggestion
  ## Default application controller's network policy
  networkPolicy:
```